src/glum/_functions.pyx

```diff
@@ -276,8 +276,8 @@ def inv_gaussian_log_rowwise_gradient_hessian(
         inv_mu = 1 / mu[i]
         inv_mu2 = inv_mu ** 2
 
-        gradient_rows_out[i] = 2 * weights[i] * (inv_mu - y[i] * inv_mu2)
-        hessian_rows_out[i] = 2 * weights[i] * (inv_mu - 2 * y[i] * inv_mu2)
+        gradient_rows_out[i] = weights[i] * (y[i] * inv_mu2 - inv_mu)
+        hessian_rows_out[i] = weights[i] * inv_mu
```

Comment:
```suggestion
        hessian_rows_out[i] = weights[i] * (2 * y[i] * inv_mu2 - inv_mu)
```

Reply:
I thought this only works for tweedies with 1 ≤ p ≤ 2, because otherwise the hessian is not positive definite ([see reference](https://github.com/Quantco/glum/blob/27b123d08f5aee157b0bc1cf3357a82cba3cc512/src/glum/_functions.pyx#L359-L361))

Reply:
Added a comment. I believe [this test case](https://github.com/Quantco/glum/blob/6f18433990bebae63af8b706ded803861819215d/tests/glm/test_distribution.py#L230) should cover the Hessian, or did you mean some other kind of test?

Reply:
I was rather thinking of a test of a fitted model against some benchmark (e.g., `glmnet`).

Reply:
Ah, right, I can add a test case for that.

Reply:
Something like [this](https://github.com/Quantco/glum/blob/b67feb429892d81c3048a09bc9257fcea2de3894/tests/glm/test_glm.py#L1273-L1344)?